### ISSUE TITLE
bug report

### ISSUE THREAD

**Issue by tafeilgh** [JohN100x1/IsekaiMod#163]
**Describe the bug**
1 .This mod may conflict with Kineticist Elements Expanded. I found many related records in the gamelog
2. channeling energy given by this class is always 1d6
**To Reproduce**

This mod may conflict with Kineticist Elements Expanded. I found many related records in the gamelog
[151.0278 - IsekaiMod]: Unpatachable Feature= 96360bedde8648a8a6762e2de41b60a5name= Disintegrating Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0288 - IsekaiMod]: Unpatachable Feature= e9cf588e2ef64fb68d0ec8c566e8b294name= Dampening Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0288 - IsekaiMod]: Unpatachable Feature= 8662ccb8dd484a2f8139d46621c641fdname= Enervating Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0298 - IsekaiMod]: Unpatachable Feature= 3ae954ad56a2497b92fada3dc493b4e1name= Pulling Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0298 - IsekaiMod]: Unpatachable Feature= e53160d091914d50bfc1d8d4fa482e30name= Unnerving Inufsion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0298 - IsekaiMod]: Unpatachable Feature= 922fd10e3e994d7793821da1583cdfeaname= Vampiric Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0307 - IsekaiMod]: Unpatachable Feature= 987dc633dbae49b0adc11cd9c5672553name= Weighing Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0317 - IsekaiMod]: Unpatachable Feature= 1cd0cd60997d4288be1fc85f753e53dename= Spore Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0317 - IsekaiMod]: Unpatachable Feature= fdc63fd61b794e40ba3c5446ba8ea1c2name= Toxic Infusion at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0327 - IsekaiMod]: Unpatachable Feature= 56567887f604473797dc8223c68999daname= Toxic Infusion, Greater at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0337 - IsekaiMod]: Unpatachable Feature= 94341284499343c0b5652f082bd677d1name= Self Telekinesis, Greater at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)
[151.0337 - IsekaiMod]: Unpatachable Feature= c03b1a6d297843c584b763378024ea7aname= Telekinetic Finesse at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)

[151.0347 - IsekaiMod]: Unpatachable Feature= 8d3e9c13037d4d4f944887b517bbf86bname= Gravity Control, Greater at level= 2 reason= Object reference not set to an instance of an object
TabletopTweaks.Core.ModLogic.ModLogger.LogError(System.String) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchClassIntoFeatureOfReferenceClass(Kingmaker.Blueprints.Classes.BlueprintFeature,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference,System.Int32,Kingmaker.Blueprints.Classes.BlueprintFeatureBase[]) (at :0)
IsekaiMod.Utilities.PatchTools.PatchProgressionFeaturesBasedOnReferenceClass(Kingmaker.Blueprints.Classes.BlueprintProgression,Kingmaker.Blueprints.BlueprintCharacterClassReference,Kingmaker.Blueprints.BlueprintCharacterClassReference) (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.KineticLegacy.PatchProgression() (at :0)
IsekaiMod.Content.Features.IsekaiProtagonist.InheritedClassFeature.LegacySelection.ConfigureStep3() (at :0)
IsekaiMod.Content.StartGameLoader_LoadAllJson.Postfix() (at :0)
Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadAllJson_Patch7(Kingmaker.Blueprints.JsonSystem.StartGameLoader,System.String) (at :0)
Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext() (at :0)
UnityEngine.SetupCoroutine.InvokeMoveNext(System.Collections.IEnumerator,System.IntPtr) (at :0)

**Expected behavior**
I want channeled energy to increase dice with level

**Comment by JohN100x1** [comment]
I healed 7 hp at level 3 using the channel energy special power.
Which channel energy are you refering to?

**Comment by tafeilgh** [comment]
I chose Dread Knight Legacy, the channel negative energy "d33eed71b9924d3880369a481eae576a" given by this class caused 3d6 damage,
The ability "3c96cd9768f34830b25b67b0199f8bb8" given by Isekai Protagonist is only 1D6
Phylactery of Negative Channeling can increase the damage caused by Dread Knight channel negative energy, but isekai channel negative energy is still 1D6.
By the way, can Magical Amplification make the channel energy dice become d10 in a future patch? Thanks

**Comment by JohN100x1** [comment]
Ok I was able to locate the issue. The scaling didn't happen for the `Channel Negative Energy` (3c96cd9768f34830b25b67b0199f8bb8) because a component wasn't patched properly.
After fixing this issue, the Phylactery of Negative Channeling wouldn't work because the ability scales by _character level_, not class/caster level - which is what the item increases. So I decided to change the ability scaling to Isekai Protagonist class level which will fix it.
Magical Amplification only works on spells, and channel energy is counted as a special ability, not a spell. I don't have plans to change how this works.
